### ISSUE TITLE
Do not recommend AdBlocker Ultimate and remove it from AMO

### ISSUE THREAD

**Issue by danhawkes** [mozilla/addons#1196]
### Describe the problem and steps to reproduce it:
This is a rehash of a previous issue that I think needs considering again: #1078 

_tldr; AdBlocker Ultimate was failing to publish its sources under GPL._

Since then, AdBlocker Ultimate has released several updates and still not published the sources as required by the license. The last source update was 10 months ago.

Rewarding addons such as this with 'recommended'  status undermines faith in the quality of the AMO review process. I would expect that manual review should require that addons purporting to be GPLv3 should make their source available _at the time of review_, and not get away with[ promising to do it later](https://github.com/mozilla/addons/issues/1078#issuecomment-515110603).

<https://addons.mozilla.org/en-GB/firefox/addon/adblocker-ultimate/versions/?src=search>
![Screenshot_2020-05-08 AdBlocker Ultimate version history - 11 versions – Add-ons for Firefox (en-GB)](https://user-images.githubusercontent.com/3388897/81392199-d4433c80-910d-11ea-91da-7437a216bda6.png)

<https://github.com/adblockultimate/AdBlocker-Ultimate-for-Firefox>
![Screenshot_2020-05-08 adblockultimate AdBlocker-Ultimate-for-Firefox](https://user-images.githubusercontent.com/3388897/81392281-f89f1900-910d-11ea-94e9-2d4c269159a8.png)

**Comment by jvillalobos** [comment]
This Github repo is for the add-ons site code, not its content. Additionally, we don't check code licensing as part of our review or recommendation process. It would actually be legally problematic if we did.

[We have a forum](https://discourse.mozilla.org/c/add-ons) where you can continue the discussion if you would like to.
